.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "aktualizr"]
 	path = aktualizr
-	url = https://github.com/foundriesio/aktualizr
```

Comment:
I think I need to create a new branch (2022.1+fio) in foundriesio/aktualizr for this change.

Reply:
I think it has to do wit this change https://github.com/uptane/aktualizr/commit/0b49e03970ef4cd20d36f0ff7388b15cf39b0313.
Let me make corresponding/adjusting changes in aklite.

Reply:
> I think I need to create a new branch (2022.1+fio) in foundriesio/aktualizr for this change.

I was wondering about that. Is that something I can do if I want to push changes like this, or does it have to be done by y'all?

> I think it has to do wit this change [uptane/aktualizr@0b49e03](https://github.com/uptane/aktualizr/commit/0b49e03970ef4cd20d36f0ff7388b15cf39b0313).

That's what I both feared and hoped.

> Let me make corresponding/adjusting changes in aklite.

Thanks! Let me know if I can help.

Reply:
> I was wondering about that. Is that something I can do if I want to push changes like this, or does it have to be done by y'all?

@doanac What do you think about it?

Reply:
@pattivacek You can refer to this branch https://github.com/foundriesio/aktualizr/tree/2022.1%2Bfio now.

Reply:
I don't mind if patti wants to push mergeups to our aktualizr  fork. Patti is more in tuned with what's happening there anyway.

Reply:
I suppose we need to grant @pattivacek with permissions to create PRs to foundriesio/aktualizr, unless it's already set.